common/FormattingUtils.js

```diff
@@ -1,6 +1,6 @@
 const BigNumber = require("bignumber.js");
 
-export function formatDate(timestampInSeconds, web3) {
+const formatDate = (timestampInSeconds, web3) => {
```

Comment:
This PR requires a fair amount of string formatting. Much of this logic will be re-used in other system monitors and in other parts of the stack. To generalize these function calls I added these formatting utils to this common file.